docs/reference/dlm/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+[role="xpack"]
+[[data-lifecycle-management]]
+== {dlm-init}: Manage your data lifecycle
+
+preview::[]
+
+{dlm-cap} ({dlm-init}) enables you to automatically manage data streams
+according to your retention requirements. For example, you could use {dlm-init} to:
+
+* Ensure that data indexed in a data stream will be kept at least for the retention time you defined.
+* Older data will be deleted by {es} any moment later in time considering {es}'s performance.
+
+You can create a data stream via index templates and configure your data lifecycle as part of the index template or you
```

Comment:
Should we talk here about the data stream lifecycle _configuration_  for new and existing data streams as opposed to creating a data stream? (it's not technically correct that you create a data stream using the index templates but more so that a write request that matches an index pattern in some templates will trigger the creation of the data stream)

Reply:
Before moving into the tutorials, should we describe some of the goals and internals of DLM? 
e.g. that it runs in a loop based on a setting, that it only applies to data streams,  it looks to roll over the write index of the data stream and to execute the retention for the other backing indices 
I guess something similar to the ILM overview https://www.elastic.co/guide/en/elasticsearch/reference/current/overview-index-lifecycle-management.html

What do you think?

Reply:
I think we should also talk here a bit about how the `lifecycle` bit of an index template is only applied to NEW data streams (as opposed to the other parts of the `template` like `settings` and `mappings` that are applied to every new _backing index_ of a data stream) i.e. on a data stream rollover the `lifecycle` will NOT be configured on the data stream

I think this is quite a fundamental difference in the properties of a `template` and we should make the behaviour very clear - I'd say here and in the index template docs

What do you think?

Reply:
I agree with all of the above, I will work on it.